websites/D/Deezer/presence.ts

```diff
@@ -2,7 +2,11 @@ const presence = new Presence({
   clientId: "607651992567021580"
 });
 
-let oldLang: string = null;
+let oldLang: string = null,
+  albumCoverURL: string,
+  albumCoverID: string,
+  podcastCoverURL: string,
+  podcastCoverID: string;
```

Comment:
Improper use of camelCase
```suggestion
  albumCoverURL: string,
  albumCoverId: string,
  podcastCoverURL: string,
  podcastCoverId: string;
```